### PR TITLE
linux_stress: stress app should not be killed

### PR DESCRIPTION
**Pull request by vivianQizhu**
Stress app should not be killed without stress_duration defined

id: 1641772
Signed-off-by: Qianqian Zhu <qizhu@redhat.com>

### CODE REVIEW

**Comment by luckyh** [comment]
@balamuruhans request for your review, thanks.

**Comment by yanan-fu** [comment]
Acked-by: Yanan Fu <yfu@redhat.com>

File: generic/tests/linux_stress.py [hunk 1]
```diff
@@ -45,24 +45,23 @@ def run(test, params, env):
 
     if stress_duration:
```
[the remark this answers is not in the document]
Comment (vivianQizhu) [reply generic/tests/linux_stress.py]: @balamuruhans Thanks for review.
As I explained in the `NOTE`, that's intended to not kill the stress app and just keep it running when no stress_duration defined. This is actually the main scenario of this linux_stress test in our testing. So we neither cancel it nor stop/clean it. 
I will correct L14, while current L26 is exactly what we need.

**Comment by luckyh** [comment]
Thanks all.

File: generic/tests/linux_stress.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (balamuruhans) [reply generic/tests/linux_stress.py]: @vivianQizhu okay, Thank you for explaining it.

**Pull request merged**